.ci/azure-pipelines/build-ubuntu-19-10.yaml

```diff
@@ -0,0 +1,53 @@
+resources:
+  containers:
+    - container: env1910
+      image: pointcloudlibrary/env:19.10
+
+jobs:
+  - job: ubuntu1910
+    displayName: Ubuntu 19.10
+    timeoutInMinutes: 0
+    pool:
+      vmImage: 'Ubuntu 16.04'
```

Comment:
is this correct?

Reply:
Yeah, this selects an agent from their pool. This agent runs requested Docker image then.